### ISSUE TITLE
HonorAssistTooltip throws error when matching MouseFocus to hardcoded string "targettargettarget"

### ISSUE THREAD

**Issue by Cjewett** [Cjewett/HonorAssist#16]
Message: Interface\AddOns\HonorAssist\HonorAssistToolTip.lua:12: attempt to index a nil value
Time: Wed Nov 27 12:45:08 2019
Count: 1
Stack: Interface\AddOns\HonorAssist\HonorAssistToolTip.lua:12: attempt to index a nil value
[C]: ?
Interface\AddOns\HonorAssist\HonorAssistToolTip.lua:12: in function <Interface\AddOns\HonorAssist\HonorAssistToolTip.lua:7>
[C]: ?
[C]: ?
...Ons\ElvUI\Libraries\Ace3\AceHook-3.0\AceHook-3.0.lua:95: in function <...Ons\ElvUI\Libraries\Ace3\AceHook-3.0\AceHook-3.0.lua:87>
[C]: ?
...Ons\BootyBayGazette\libs\LibExtraTip\LibExtraTip.lua:482: in function <...Ons\BootyBayGazette\libs\LibExtraTip\LibExtraTip.lua:476>
[C]: ?
[C]: ?
...Ons\BootyBayGazette\libs\LibExtraTip\LibExtraTip.lua:408: in function <...Ons\BootyBayGazette\libs\LibExtraTip\LibExtraTip.lua:402>
[C]: in function `SetHyperlink'
Interface\AddOns\VanasKoS\modules\WarnFrame.lua:145: in function <Interface\AddOns\VanasKoS\modules\WarnFrame.lua:134>
Interface\AddOns\VanasKoS\modules\WarnFrame.lua:345: in function <Interface\AddOns\VanasKoS\modules\WarnFrame.lua:344>

Locals:

_Originally posted by @dagochen in https://github.com/Cjewett/HonorAssist/pull/15#issuecomment-559054239_

**Comment by Cjewett** [comment]
From @dagochen
everything else should work fine for me

**Comment by Cjewett** [comment]
From @mplachter 
@dagochen are you saying you pulled this branch and tried it and it was all working well minus the tooltip hover? And yeah thats not working since it doesn't have localization i believe.

**Comment by Cjewett** [comment]
From @mplachter 
Can you do me a favor and do a /fstack and hover over your target or target of target and get me a spinet of what it displays as.

**Comment by Cjewett** [comment]
From @dagochen
i ll do it tomoroww ;)

**Comment by Cjewett** [comment]
@mplachter @dagochen If I had to guess GetMouseFocus() is most likely nil in certain scenarios, or whatever mouse focus is does not have a name.

**Comment by mplachter** [comment]
That could be the case. We can add a condition to make sure the frame has a name. Thoughts?

**Comment by Cjewett** [comment]
Yeah, should also add some debug logging just so we can turn it on and figure out the exact cause.
